### PR TITLE
chore: standard object selection for gong

### PR DESCRIPTION
**Pull request by albertyfwu**
## Test Plan

[Describe test plan here]

## Deployment instructions

[Add any special deployment instructions here]

### CODE REVIEW

File: apps/mgmt-ui/src/components/configuration/syncConfig/SyncConfigDetailsPanel.tsx [hunk 1]
```diff
@@ -233,7 +233,7 @@ function SyncConfigDetailsPanelImpl({ syncConfigId }: SyncConfigDetailsPanelImpl
                   key={providerId}
                   multiple
                   id="common-objects"
-                  options={CRM_COMMON_OBJECT_TYPES}
+                  options={provider.category === 'crm' ? CRM_COMMON_OBJECT_TYPES : ENGAGEMENT_COMMON_OBJECT_TYPES}
```
Comment (albertyfwu) [inline apps/mgmt-ui/src/components/configuration/syncConfig/SyncConfigDetailsPanel.tsx:236]: unrelated bug fix

**Review by lucasmarshall — APPROVED** [review]
